packages/fxa-event-broker/README.md

```diff
@@ -70,7 +78,7 @@ Services should update any cached profile data they hold about the user.
      "jti": "e19ed6c5-4816-4171-aa43-56ffe80dbda1",
      "events": {
        "https://schemas.accounts.firefox.com/event/profile-change": {
-         "email": "example@mozilla.com"
+         "uid": "cd1181e0532c45cb989a7c234641468e"
```

Comment:
The UID was always being sent, and in a couple cases the email was being sent too, but in most cases it was not sent. We need to follow up with RPs about this change.